### ISSUE TITLE
Add Image Caption

### ISSUE THREAD

**Issue by freinhard** [OneGov/onegov.town#82]
Images should have an optional image caption. The text is added via the redactor.js image plugin (title) and saved in the HTML field.

![image](https://cloud.githubusercontent.com/assets/186856/10330581/58e66b92-6cce-11e5-96cb-21feaaa42edb.png)

**Comment by href** [comment]
This should be done by simply showing the alt text using css

**Comment by msom** [comment]
Pure css won't do it, :before / :after are not supported on img tags.

**Comment by href** [comment]
Awww. I guess javascript is the next best option here... Otherwise we have to add the element to the html internally and remove it again when showing it in the html editor.

**Comment by msom** [comment]
+1

**Comment by href** [comment]
Just be sure to only get the images inside the page content, not *all* images should have this added element

**Comment by msom** [comment]
Maybe we should better use <figure> and <figcaption> for semantic reasons (although for older browsers shim is required and additional markup)...

**Comment by href** [comment]
+1, though let's not do that until the more pressing issues are resolved.
